.codecov.yml

```diff
@@ -0,0 +1,8 @@
+coverage:
+  status:
+    project: #add everything under here, more options at https://docs.codecov.com/docs/commit-status
+      default:
+        # basic
+        target: auto #default
+        threshold: 35%
```

Comment:
does 35% apply to new added code?

Reply:
It's a threshold.
Now the overall UT coverage of the Kmesh is36.06%.
If a new commit causes project coverage to fall below 35%, an alarm is raised.